vng_api_common/utils.py

```diff
@@ -148,3 +148,18 @@ def get_help_text(model_string: str, field_name: str) -> str:
     ModelClass = apps.get_model(model_string, require_ready=False)
     field = ModelClass._meta.get_field(field_name)
     return field.help_text
+
+
+def notification_documentation(kanaal):
+    """
+    Generate notification documentation for an OpenAPI specification containing
+    the relevant resources and actions for a given KANAAL
+    """
+    doc = "### Notificaties\n\n"
```

Comment:
I think I prefer a triple-quote multiline f-string for readability instead of the `+=` operations.